examples/with-typescript-recommended/src/preconditions/OwnerOnly.ts

```diff
@@ -1,13 +1,22 @@
 import { Precondition } from '@sapphire/framework';
```

Comment:
Look at https://github.com/favware/dragonite/blob/main/src/preconditions/OwnerOnly.ts for a proper impl

- Extend AllFlowsPrecondition
- Do not duplicate code